src/pages/TalkPickPage/TalkPickPage.tsx

```diff
@@ -24,6 +25,7 @@ const TodayTalkPickPage = () => {
   const location = useLocation();
   const state = location.state as State;
   const talkPickId = state?.talkPickId;
+  const isTodayTalkPick = state?.isTodayTalkPick;
```

Comment:
```suggestion
const { talkPickId } = useParams<{ talkPickId: string }>();
  const isTodayTalkPick = state?.isTodayTalkPick ?? false;
```

이런 식으로 URL 매개변수는 useParams를 이용해 가져오는 방향도 괜찮아 보여요! 
두번째 줄의 경우는 `const isTodayTalkPick = !talkPickId;` 처럼 다양한 방식으로 표현할 수도 있을 거 같아요!

라우팅 상에서 매개변수를 가져가고 있어서 useParams 을 이용한 방식도 추천 드립니다 :)

Reply:
단순한 데이터 전달이고 url 내에 포함되어 있으니 좋은 방식인거 같습니다! 하지만 `/todaytalkpick` 으로 이동하는 경우에는 해당 url에 talkpickId값이 존재하지 않아 어차피 `useLocation`을 사용해야하기 때문에 공통적으로 사용해주었는데, 이 부분도 useParams로 수정할 수 있는 방법이 있을까요?

Reply:
사실 현재 구현되어있는 방식으로는 하나의 page에서 `TodayTalkPickPage` 와 일반 `TalkPickPage` 를 출력해야 하기 때문에 말씀해주신 것처럼 결국 `useLocation`은 필수적인 것 같네요! 

라우팅 주소가 아예 다르기 때문에 현재 방식으로 유지해도 문제 없을 것 같습니다ㅎㅎㅎ